<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>7</number>
        </property>
        <item>
         <widget class="QPushButton" name="ndpi_exporter">
          <property name="minimumSize">
           <size>
            <width>208</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>Whole Slide Image Exporter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testingbutton">
          <property name="minimumSize">
           <size>
            <width>208</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>Optimise and Set Thresholds</string>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trainingbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>60</horstretch>
            <verstretch>60</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>208</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>Run DAB Analysis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="overbtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>60</horstretch>
            <verstretch>60</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>208</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>Overlay Figure Creator (beta)</string>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Save Images</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>DAB analyser</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Created by - Callum Arthurs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Version 0.0.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>For analysis of NDPI and SVS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>whole slide images, based on</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>analysis pipeline developed in</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>the Ahmed Lab - KCL.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Images should be exported</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>via the NDPI exporter and </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>analysed in the application.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>For bug reports, email - </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>callum.arthurs@kcl.ac.uk</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QProgressBar" name="progress">
        <property name="minimumSize">
         <size>
          <width>163</width>
          <height>0</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNDPI_SVS_Export"/>
    <addaction name="actionRun_Dab_Analysis"/>
    <addaction name="actionSet_Thresholds"/>
    <addaction name="actionOverlay_Figure"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRun_Dab_Analysis">
   <property name="text">
    <string>Run Dab Analysis</string>
   </property>
  </action>
  <action name="actionSet_Thresholds">
   <property name="text">
    <string>Set Thresholds</string>
   </property>
  </action>
  <action name="actionOverlay_Figure">
   <property name="text">
    <string>Overlay Figure</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNDPI_SVS_Export">
   <property name="text">
    <string>NDPI/SVS Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
